packages/trie/package.json

```
@@ -1,5 +1,5 @@
 {
-  "name": "@ethereumjs/trie",
+  "name": "merkle-patricia-tree",
   "version": "4.0.0",
   "description": "This is an implementation of the modified merkle patricia tree as specified in the Ethereum's yellow paper.",
```

just noticed this nit :)
```suggestion
  "description": "This is an implementation of the modified merkle patricia tree as specified in Ethereum's yellow paper.",
```